packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -26,6 +26,10 @@ import { getWarningForWorkersConfigs } from './workers-configs';
 import type { PluginConfig, ResolvedPluginConfig } from './plugin-config';
 import type { Unstable_RawConfig } from 'wrangler';
 
+/**
+ * Cloudflare Vite plugin.
+ * Accepts an optional {@link PluginConfig} object.
```

Comment:
Should we have a more detailed description?

For the parameter, why not a standard [`@param` tag](https://jsdoc.app/tags-param)? 😮

Reply:
To be honest, I just followed Vite's example here - https://github.com/vitejs/vite/blob/4f5845a3182fc950eb9cd76d7161698383113b18/packages/vite/src/node/config.ts#L137-L141.

Feel free to make a suggestion.

Reply:
Thanks. Let's add the JSDoc once the readme is in workers-sdk so it points to the right place.